ligo/main.mligo

```diff
@@ -38,13 +38,18 @@ let rec initialize_tick ((ticks, i, i_l,
         else
             initialize_tick (ticks, i, i_next, initial_fee_growth_outside, initial_seconds_outside, initial_seconds_per_liquidity_outside)
 
-let incr_n_positions (ticks : tick_map) (i : tick_index) (incr : int) =
+(* Account that this tick is a boundary for one more (or one less) position. *)
```

Comment:
```suggestion
(* Account for the fact that this tick is a boundary for one more (or one less) position. *)
```

Reply:
Did you accidentally undo the changes to `incr_n_positions` during a rebase? Or was that on purpose? :open_mouth:

Reply:
Oh I see this has been moved to #11, nevermind :sweat_smile:

Reply:
Yeah :pled: :smile_cat: